### ISSUE TITLE
Unreal! Computer hangs for over 2-3 minutes!

### ISSUE THREAD

**Issue by RowanRishi** [jgraph/drawio-desktop#1151]
### Computer hangs for over 2-3 minutes!

Hey guys! Is this a bug? I'll keep this short and sweet. I am using draw.io to make a plan of humongous size. It's HUGE. The problem is that if I change anything on my diagram, like the zoom level, draw.io hangs for a very long time. I have a work-around for this: I always increase the CPU priority from "Normal" to "Real Time" - then the wait time decreases to 1-2 minutes.

Also, by checking in my task manager, I noticed that only 1 core was being used - and 7 other cores were just being idle!

In short -

1) Use all my CPU resources.
2) Work only to render what's needed.

This will be a game changer. I pray that my requests are not too hard to implement!

-Rowan
